tesseract_core/sdk/templates/docker-compose.yml

```diff
@@ -14,8 +14,8 @@ services:
     {% endif %}
     {%- if service.volumes %}
     volumes:
-    {%- for volume in service.volumes %}
-      - {{ volume }}
+    {%- for volume_source, volume_data in service.volumes.items() %}
+      - {{ volume_source }}:{{ volume_data["bind"] }}
```

Comment:
Does this work for named volumes as well?

Reply:
yes, see tests